### DISCUSSION TITLE
[Request] Adding mini-player back on 'made for kids' videos

### DISCUSSION THREAD

**Discussion by FizzyMcD** [qnblackcat/uYouPlus#877] [Ideas]
Hey!
YouTube's removal of mini-player on 'made for kids' videos is infuriating, it would be amazing if videos weren't auto-paused when browsing the app.

**Comment by yodaluca23** [comment]
> Hey!
> 
> YouTube's removal of mini-player on 'made for kids' videos is infuriating, it would be amazing if videos weren't auto-paused when browsing the app.

Their is the option in 
Settings>uYou+>Video Player Controls>Enable Mini-Player for all YouTube Videos
